parlai/agents/transformer/transformer.py

```diff
@@ -49,6 +52,11 @@ def add_common_cmdline_args(argparser):
     argparser.add_argument('--activation', choices={'relu', 'gelu'}, default='relu',
                            help='Nonlinear activation to use. AIAYN uses relu, but '
                                 'more recent papers prefer gelu.')
+    argparser.add_argument('--output-scaling', type=float, default=1.0,
+                           help='scale the output of every transformer by this quantity.')
+    argparser.add_argument('--share_word_embeddings', type='bool', default=True,
```

Comment:
nit -- we've been using `-` instead of `_`

Reply:
thx